src/scripts/visualizations/tooltip.js

```diff
@@ -112,6 +124,12 @@
             changeOpacity.call(this, 0, this.options.tooltip.hideTime);
         };
 
+        var onMouseMove = function (d) {
+            if (this.options.tooltip.followCursor) {
+                show.call(this, d)
```

Comment:
I think so, need the "this" in the show function. Alternatively can do ```show.bind(this)(d)``` but I think call is better.